app/main.py

```diff
@@ -2,24 +2,84 @@
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: type, name: str) -> None:
+        self.public_name = name
+        self.protected_name = "_" + name
+
+    def __get__(self, instance: object, owner: type) -> int:
+        return getattr(instance, self.protected_name)
+
+    def __set__(self, instance: object, value: int) -> None:
+        if self.min_amount <= value <= self.max_amount:
+            setattr(instance, self.protected_name, value)
+            if not hasattr(instance, "access"):
+                instance.access = True
+        else:
+            print(f"{instance.slide_name}: access is not granted"
+                  f" because of {self.public_name}({value}) "
+                  f"is not in range({self.min_amount}-{self.max_amount})")
+            if not hasattr(instance, "access") or instance.access:
+                instance.access = False
 
 
 class Visitor:
-   pass
+    def __init__(self,
+                 name: str,
+                 age: int,
+                 weight: int,
+                 height: int
+                 ) -> None:
```

Comment:
Place `self` on a new line here to match the code style when the opening and closing parenthesis do not have arguments on the same lines with them. Apply this change to all classes where applicable.